src/custom_nodes/east_ocr/east_ocr.cpp

```diff
@@ -407,19 +403,19 @@ int getInputsInfo(struct CustomNodeTensorInfo** info, int* infoCount, const stru
     (*info)[1].dims = (uint64_t*)malloc((*info)->dimsCount * sizeof(uint64_t));
     NODE_ASSERT(((*info)[1].dims) != nullptr, "malloc has failed");
     (*info)[1].dims[0] = 1;
-    (*info)[1].dims[1] = 1;
```

Comment:
This custom node will not work with setups prior v2022.1. Should we notify customers?